cfme/modeling/base.py

```diff
@@ -199,6 +199,21 @@ def delete_if_exists(self, *args, **kwargs):
                 raise NotImplementedError("Delete method is not implemented.")
         return False
 
+    def expected_summary_attributes(self):
```

Comment:
I think it would be better to create two properties for `entity.expected_title` and `entity.expected_breadcrumb`. I'm not 100% on those names, maybe `summary_<thing>`, or `details_<thing>`.

Whatever it is, we want to be able to make it consistent across all entities to access common entity-based expected strings. Example - collection title/breadcrumb, CRUD method page titles.